web_poet/testing/__init__.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+from pathlib import Path
+from typing import Any, Iterable, Union
+
+from itemadapter import ItemAdapter
+
+from web_poet.serialization import SerializedDataFileStorage, serialize
+
+INPUT_DIR_NAME = "inputs"
+OUTPUT_FILE_NAME = "output.json"
+
+
+def save_fixture(
+    base_directory: Union[str, os.PathLike], inputs: Iterable[Any], item: Any
+) -> None:
+    inputs_dir = Path(base_directory, INPUT_DIR_NAME)
+    inputs_dir.mkdir(parents=True)
+    serialized_inputs = serialize(inputs)
+    storage = SerializedDataFileStorage(inputs_dir)
+    storage.write(serialized_inputs)
+    with Path(base_directory, OUTPUT_FILE_NAME).open("w") as f:
+        json.dump(ItemAdapter(item).asdict(), f)
```

Comment:
let's use ensure_ascii=False. Not sure about sort_keys=True, probably makes sense not to use it.

Reply:
I'd also use some indent option here, indent=2 or indent=4.